sw/airborne/modules/nav/common_flight_plan.h

```diff
@@ -60,4 +60,7 @@ void nav_goto_block(uint8_t block_id);
 /** Time in s since the entrance in the current block */
 #define NavBlockTime() (block_time)
 
+/** macro to use pointers in flight plans XML */
+#define Ptr(x) (&(x))
```

Comment:
where is this used?

Reply:
In the rotorcraft flight plan, the XML parser don't like the `&` symbol